app/actions/remote/entry/common.ts

```diff
@@ -360,6 +363,51 @@ const syncAllChannelMembersAndThreads = async (serverUrl: string) => {
         return;
     }
 
+    const config = await getConfig(database);
+
+    if (config?.FeatureFlagGraphQL === 'true') {
+        const error = await graphQLSyncAllChannelMembers(serverUrl);
+        if (error) {
+            logDebug('failed graphQL, falling back to rest', error);
+            restSyncAllChannelMembers(serverUrl);
+        }
+    } else {
+        restSyncAllChannelMembers(serverUrl);
+    }
+};
+
+const graphQLSyncAllChannelMembers = async (serverUrl: string) => {
+    const response = await gqlAllChannels(serverUrl);
+    if ('error' in response) {
+        return response.error;
+    }
+
+    if (response.errors) {
+        return response.errors[0].message;
+    }
+
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        return 'Server database not found';
+    }
+
+    const userId = await getCurrentUserId(operator.database);
+
+    const channels = response.data.channelMembers?.map((m) => gqlToClientChannel(m.channel!));
```

Comment:
can you guarantee that `channel` is available at all times in this context ? asking cause of the `!`

Reply:
In theory, it should be defined, because if not, graphQL would have thrown an error, but I added a safeguard, just in case.